src/common/Journald.h

```diff
@@ -0,0 +1,70 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef __CEPH_LOG_JOURNALD_H
```

Comment:
please avoid using names starting with two underlines. they are reserved for compiler and libraries. see https://gcc.gnu.org/onlinedocs/cpp/System-specific-Predefined-Macros.html